### ISSUE TITLE
Hessian Matrix

### ISSUE THREAD

**Issue by kingofthewinds** [mml-book/mml-book.github.io#363]
Version (2019-07-25), Chapter 5, Section 5.7: 
By definition, the hessian in equation 5.147 is always symmetric, I suggest you first properly define the hessian (like in 5.171) and then explain how twice differentiable functions make it symmetric. 

The book is really good, I like it :)

**Comment by mpd37** [comment]
Good point. Unfortunately, we can't make these (significant) changes at this stage anymore.
